receiver/apachereceiver/metadata.yaml

```diff
@@ -76,6 +76,129 @@ metrics:
       monotonic: true
       aggregation: cumulative
     attributes: [ server_name ]
+  apache.bytes_per_request:
+    enabled: true
+    description: The average number of bytes per request.
+    unit: By
+    sum:
+      value_type: double
+      input_type: string
+      monotonic: false
+      aggregation: delta
```

Comment:
As notes above, I don't think this metric is actually a sum. For gauges, `monotonic` and `aggregation` are not applicable.

That said, it may be useful to explain these further, as I think I'll have related feedback below.

I find it easiest to explain with two examples:
1. A metric that reports the number of requests made to and endpoint.
2. A metric that reports the current number of users logged into a service.

In the "requests" case, each request causes the value to go up. This is monotonic.
In the "current users" case, each time a user logs in the value goes up, but each time a user logs out it goes down. This is nonmonotonic.

Note that the monotonicity of the measurements are inherently monotonic or not. On the other hand, either metric can be presented as cumulative or delta aggregation.

Suppose each metric is reported once per minute, for five consecutive minutes:
| scenario | monotonic | aggregation | example | representation |
| - | - | - | - | - |
| requests | true | cumulative | 5 requests per minute | [ 5, 10, 15, 20, 25, ... ] |
| requests | true | delta | 5 requests per minute | [ 5, 5, 5, 5, 5, ... ] |
| current users | false | cumulative | 5 log in, then 1 logs out, repeated | [ 5, 4, 9, 8, 13, 12, ... ] |
| current users | false | delta | 5 log in, then 1 logs out, repeated | [ 5, -1, 5, -1, 5, -1, ... ] |

Reply:
Thank you for this explanation! I'm relatively new to OpenTelemetry and I'm not used to the terminology yet.